install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -957,6 +957,8 @@ gatewayAPI:
   # Note that this value will be ignored when `hostNetwork.enabled == true`.
   # ref: https://kubernetes.io/docs/reference/networking/virtual-ips/#external-traffic-policy
   externalTrafficPolicy: Cluster
+  # -- Control the dual-stack-ness requested or required by the LoadBalancer Kubernetes Service for all Cilium GatewayAPI Gateway instances. Valid values are "SingleStack", "PreferDualStack" and "RequireDualStack"
+  ipFamilyPolicy: SingleStack
```

Comment:
Does it make sense for this to default to PreferDualStack if both families are enabled?

Reply:
I wanted to stay as close as possible to the current behavior.

Reply:
gotcha, seems reasonable.